### PR TITLE
Adds Breadcrumbs component to Club and Event components

### PR DESCRIPTION
**Pull request by cooperama**
Creates Breadcrumbs component and adds to Club and Event components.

### CODE REVIEW

File: src/pages/GetInvolved/Club.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ import React from "react";
 import { useSelector } from "react-redux";
 import { withRouter, Link } from "react-router-dom";
 
+import Breadcrumbs from "../../components/Breadcrumbs/Breadcrumbs";
```
Comment (mattlaguardia) [inline src/pages/GetInvolved/Club.js:17]: No need for relative path

```suggestion
import Breadcrumbs from "components/Breadcrumbs/Breadcrumbs";
```

File: src/pages/Event/Event.js [hunk 2]
```diff
@@ -21,9 +22,7 @@ const Event = ({ match }) => {
   return (
     <Layout>
       <section className="event">
-        <div className="event__breadcrumbs">
-          <Link to='/'>Home</Link> / <Link to='/events'>Events</Link> / <span className="event__highlight">{ event_title }</span>
-          </div>
+      <Breadcrumbs parent="events" parentText="Events" rendered={ event_title } />
```
Comment (mattlaguardia) [inline src/pages/Event/Event.js:25]: Could you delete the styling for this in the events.scss file too?

File: src/components/Breadcrumbs/Breadcrumbs.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    &--selected {
+      color: $primary-1;
+    }
+  }
+
+  &__item:hover {
+    color: $primary-1;
+  }
+  
+}
```
Comment (mattlaguardia) [inline src/components/Breadcrumbs/Breadcrumbs.scss:16]: Github likes empty lines at the bottom of files. Could you config your editor to do this?

Screen Shot 2021-01-25 at 1 37 07 PM


```suggestion
}

```

File: src/components/Breadcrumbs/Breadcrumbs.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    /
+    <Link className="breadcrumb__item" to={ `/${parent}` } >
+      { parentText }
+    </Link>
+    /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
+  </p>
+  )
+}
+
+export default Breadcrumbs;
```
Comment (mattlaguardia) [inline src/components/Breadcrumbs/Breadcrumbs.js:19]: Github likes empty lines at the bottom of files. Could you config your editor to do this?

Screen Shot 2021-01-25 at 1 38 36 PM



```suggestion
export default Breadcrumbs;

```

File: src/components/Breadcrumbs/Breadcrumbs.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    <Link className="breadcrumb__item" to="/">
+      Home
+    </Link>
+    /
+    <Link className="breadcrumb__item" to={ `/${parent}` } >
+      { parentText }
+    </Link>
+    /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
+  </p>
+  )
```
Comment (mattlaguardia) [inline src/components/Breadcrumbs/Breadcrumbs.js:16]: Spacing seems a bit off...

```suggestion
  return (
    <p className="breadcrumb">
      <Link className="breadcrumb__item" to="/">
        Home
      </Link>
      /
      <Link className="breadcrumb__item" to={ `/${parent}` } >
        { parentText }
      </Link>
      /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
    </p>
  )
```

File: src/components/Breadcrumbs/Breadcrumbs.scss [hunk 6]
```diff
@@ -0,0 +1,16 @@
+.breadcrumb {
+  margin-top: 6vh;
```
Comment (mattlaguardia) [inline src/components/Breadcrumbs/Breadcrumbs.scss:2]: Could this be converted to `rem` or `px`, `vh` is relative to the screen size which we probably don't want

Comment (cooperama) [reply src/components/Breadcrumbs/Breadcrumbs.scss]: I just left the styles as I found them, but looking at the figma board now, it looks like the breadcrumbs should be a little higher than they are. If I remove the margin-top it seems to look about even with the sidebar. Should I maybe just remove this line in that case?

File: src/components/Builder/Breadcrumbs/Breadcrumbs.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    <p className="breadcrumb">
+      <Link className="breadcrumb__item" to="/">
+        Home
+      </Link>
+      /
+      <Link className="breadcrumb__item" to={ `/${parent}` } >
+        { parentText }
+      </Link>
+      /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
+  </p>
```
Comment (mattlaguardia) [inline src/components/Builder/Breadcrumbs/Breadcrumbs.js:15]: spacing is still off on the closing p tag
```suggestion
    <p className="breadcrumb">
      <Link className="breadcrumb__item" to="/">
        Home
      </Link>
      /
      <Link className="breadcrumb__item" to={ `/${parent}` } >
        { parentText }
      </Link>
      /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
    </p>
```

File: src/components/components.scss [hunk 8]
```diff
@@ -12,4 +12,5 @@
   'Navbar/Navbar',
   'Sidebar/Sidebar',
   'TextArea/TextArea',
+  'Builder/Breadcrumbs/Breadcrumbs',
```
Comment (mattlaguardia) [inline src/components/components.scss:15]: Lets not put this component in the Builder directory, lets just have it live in the components

**Review by mattlaguardia — APPROVED** [review]
Looks good

**Pull request merged**